src/admin/js/user/reducer.js

```diff
@@ -1,14 +1,12 @@
 import { userActionTypes } from './actions';
 
 export default function (localStorage) {
-    const notExpired = localStorage.getItem('expires') > (new Date()).getTime();
-    const authenticated = !!localStorage.getItem('token') && notExpired;
+    const authenticated = !!localStorage.getItem('token');
```

Comment:
Why remove the frontend check verification?
It will save a request to the API and avoid a redirection to the final user.